tools/ci_build/github/azure-pipelines/templates/stages/mac-ios-packaging-build-stage.yml

```diff
@@ -117,16 +117,70 @@ stages:
         useXcpretty: false  # xcpretty can hide useful error output so we will disable it
       displayName: 'Build App Center iPhone arm64 tests'
 
+    - script: |
+        zip -r --symlinks $(Build.ArtifactStagingDirectory)/package_tests.zip ios_package_testUITests-Runner.app
+      workingDirectory: '$(Build.BinariesDirectory)/app_center_test/apple_package_test/DerivedData/Build/Products/Debug-iphoneos'
+      displayName: "Create .zip file of the tests"
+
+    - script: |
+        python $(Build.SourcesDirectory)/tools/python/generate_plist.py --dest_file "exportOptions.plist"
+      workingDirectory: '$(Build.BinariesDirectory)/app_center_test/apple_package_test/'
+      displayName: "Generate .plist file for the .ipa file"
+      env:
+        PROVISIONING_PROFILE_UUID: '$(APPLE_PROV_PROFILE_UUID)'
+        APPLE_TEAM_ID: '$(APPLE_TEAM_ID)'
+
+    - task: Xcode@5
+      inputs:
+        actions: 'archive'
+        xcWorkspacePath: '$(Build.BinariesDirectory)/app_center_test/apple_package_test/apple_package_test.xcworkspace'
+        packageApp: true
+        archivePath: '$(Build.BinariesDirectory)/app_center_test/apple_package_test/'
+        exportOptions: 'plist'
+        exportOptionsPlist: '$(Build.BinariesDirectory)/app_center_test/apple_package_test/exportOptions.plist'
+        configuration: 'Debug'
+        sdk: 'iphoneos'
+        scheme: 'ios_package_test'
+        xcodeVersion: 'specifyPath'
+        xcodeDeveloperDir: '/Applications/Xcode_${{ variables.xcodeVersion }}.app/Contents/Developer'
+        args: '-derivedDataPath $(Build.BinariesDirectory)/app_center_test/apple_package_test/DerivedData'
```

Comment:
I swapped some of these to relative paths, but for the options that request a directory, I left as absolute paths because I think that that's easier to read for later steps (ie, knowing the full path for DerivedData which is used later to find the build products)

Reply:
update: added with comment about using absolute paths